src/components/SignUp.tsx

```diff
@@ -0,0 +1,75 @@
+import {Button, Card, Help, InputField, LogoIcon} from '@dhis2/ui'
+import {useCallback, useState} from 'react'
+import {useNavigate} from 'react-router-dom'
+import {postSignUp} from '../api'
+import styles from './SignUpPage.module.css'
+
+const SignUpPage = () => {
+    const navigate = useNavigate()
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+    const [signUpError, setSignUpError] = useState('')
+
+    const doSignUp = useCallback(async () => {
+        try {
+            const result = await postSignUp(username, password)
+            if (result.status === 201) {
+                window.requestAnimationFrame(() => navigate('/login'))
+            }
+        } catch (error) {
+            setSignUpError(error.response.data)
+        }
+    }, [username, password, navigate])
+
+    return (
+        <form
+            className={styles.container}
+            action="login"
+            autoComplete="on"
+            onSubmit={(event) => {
+                event.stopPropagation()
+                event.preventDefault()
+                doSignUp()
+            }}
+        >
+            <Card className={styles.box}>
+                <h2 className={styles.header}>
+                    <LogoIcon className={styles.logo}/>
+                    Instance manager sign up
+                </h2>
+                <InputField
+                    name="username"
+                    label="username"
+                    value={username}
+                    onChange={({value}) => {
+                        setUsername(value)
+                    }}
+                />
+                <InputField
+                    type="password"
+                    name="password"
+                    label="password"
+                    value={password}
+                    onChange={({value}) => {
+                        setPassword(value)
+                    }}
+                />
+                {signUpError && <Help error>{signUpError}</Help>}
```

Comment:
Typically, you'd also have a "repeat/confirm password" field in a signup form. Should we add one too?

Reply:
I added another field to the form and tried to do some validation. Please let me know what you think :pray: